src/core/sys/posix/arpa/inet.d

```diff
@@ -199,23 +199,4 @@ NOTE: The following must must be defined in core.sys.posix.arpa.inet to break
 INET6_ADDRSTRLEN // from core.sys.posix.netinet.in_
 */
 
-version( linux )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( OSX )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( FreeBSD )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( Solaris )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
-else version( Android )
-{
-    enum INET6_ADDRSTRLEN = 46;
-}
+enum INET6_ADDRSTRLEN = 46;
```

Comment:
From doing the same google search, I see webpages that state that it's defined to be 46 in both the Posix and IPv6 standards.  I cannot tell what that platform is where it's 48, can you?  If not, there's no point in dealing with it.

I think avoiding unnecessarily redundant definitions definitely "improves the state of affairs," particularly for future porters to new platforms.  The question of whether a set of repeated definitions _is_ unnecessary will have to be determined on a case-by-case basis.

The previous code _would not_ give you an undefined identifier error if it was 48, as that header you linked appears to be for linux, which is one of the cases in the list.  So it would have silently failed previously, as it would now.  Nothing has changed with this pull for the case you bring up.

Speculatively separating declarations with no reason given is premature optimization, and we all know what that's the root of, right? ;)